### DISCUSSION TITLE
[Feature] Link "back" of photo to photo

### DISCUSSION THREAD

**Discussion by SNR-Tech-Bytes** [immich-app/immich#14422] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I am using immich to archive thousands of scanned photos from my grandmothers house to share with our family online. I'm using an Epson scanner to scan these photos and it has this cool feature where if it detects handwriting on the back of an image, it labels it as [image_name]_b.jpg indicating its the back of a photo. If possible, it would be an amazing tool to have immich automatically link these two images together! Bonus points if it hides the _b from the main timeline and only shows it when you click on a specific image to declutter the homepage It would save a ton of time for us archivists as well who have to manually dig them up and add them to the context text as well. 

### Platform

- [x] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can select both assets and stack them together. Doing that automatically will be covered by #1673/the workflows feature.
